### PR TITLE
[WIP] Marketing pages: styling + fixes broken link

### PR DESCRIPTION
**Pull request by tarafanlin**
Summary
-  styling tweaks for landing and download page
- fix broken link to slates

Next step
- add chrome extension UI to download page
- continue on the community page after Will

### CODE REVIEW

File: pages/_/marketing-candidate/download.js [hunk 1]
```diff
@@ -16,83 +16,60 @@ const STYLES_ROOT = css`
...
+  font-size: ${Constants.typescale.lvl5};
+  line-height: 1.3;
+  padding: 0px 0px 24px 0px;
+  letter-spacing: -0.021rem;
   width: 100%;
-  color: ${Constants.system.pitchBlack};
-  @media (max-width: ${Constants.sizes.mobile}px) {
-    font-size: 1.953rem;
+  color: ${Constants.system.slate};
+  font-weight: 800;
```
Comment (jimmylee) [inline pages/_/marketing-candidate/download.js:25]: We can't use font-weight here, please use `font-family`.  Please change this to `font-weight: 400` and find the appropriately font-family you need for this, It should already be in `constants`.

File: pages/_/marketing-candidate/download.js [hunk 2]
```diff
@@ -16,83 +16,60 @@ const STYLES_ROOT = css`
...
-  :active {
-    color: yellow;
-    background-color: transparent;
-    text-decoration: none;
-  }
+const STYLES_TEXT_BLOCK = css`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 56vw;
```
Comment (jimmylee) [inline pages/_/marketing-candidate/download.js:57]: try sticking to this strategy:

```js
max-width: someValuepx
width: 100%;
```

I think `56vw` provides inconsistent results.

File: pages/index.js [hunk 3]
```diff
@@ -9,13 +9,28 @@ import WebsitePrototypeFooter from "~/components/core/NewWebsitePrototypeFooter"
 import TextLoop from "react-text-loop";
 import { css } from "@emotion/react";
 
+<<<<<<< Updated upstream
```
Comment (jimmylee) [inline pages/index.js:12]: This will throw, did you test this?

File: pages/index.js [hunk 4]
```diff
@@ -9,13 +9,28 @@ import WebsitePrototypeFooter from "~/components/core/NewWebsitePrototypeFooter"
...
 import ReactDOM from "react-dom";
+=======
```
Comment (jimmylee) [inline pages/index.js:14]: This will also throw, did you test this?

File: pages/index.js [hunk 5]
```diff
@@ -9,13 +9,28 @@ import WebsitePrototypeFooter from "~/components/core/NewWebsitePrototypeFooter"
...
+  overflow-y: scroll;
+  overflow-x: hidden;
+  background-color: ${Constants.system.pitchBlack};
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+`;
+>>>>>>> Stashed changes
```
Comment (jimmylee) [inline pages/index.js:28]: This line will also throw.

File: pages/index.js [hunk 6]
```diff
@@ -123,7 +136,11 @@ const STYLES_SECTION_SLATE = css`
   flex-direction: column;
   padding: 88px;
   width: 100vw;
+<<<<<<< Updated upstream
```
Comment (jimmylee) [inline pages/index.js:139]: ✂️

File: pages/index.js [hunk 7]
```diff
@@ -123,7 +136,11 @@ const STYLES_SECTION_SLATE = css`
...
   ${"" /* background: ${Constants.system.slate}; */}
+=======
```
Comment (jimmylee) [inline pages/index.js:141]: ✂️

File: pages/index.js [hunk 8]
```diff
@@ -123,7 +136,11 @@ const STYLES_SECTION_SLATE = css`
...
+  background: ${Constants.system.pitchBlack};
+>>>>>>> Stashed changes
```
Comment (jimmylee) [inline pages/index.js:143]: ✂️

File: pages/index.js [hunk 9]
```diff
@@ -293,10 +329,13 @@ const STYLES_SLATE_CARD = css`
   width: calc(100% / 3 + 1px);
   height: calc(100vh / 4);
   margin-left: -1px;
-  border-radius: 4px;
   transition: 200ms ease box-shadow;
+<<<<<<< Updated upstream
```
Comment (jimmylee) [inline pages/index.js:333]: ✂️

File: pages/index.js [hunk 10]
```diff
@@ -293,10 +329,13 @@ const STYLES_SLATE_CARD = css`
...
   box-shadow: 0px 4px 8px 4px rgba(0, 0, 0, 0.02);
 
+=======
```
Comment (jimmylee) [inline pages/index.js:336]: ✂️

File: pages/index.js [hunk 11]
```diff
@@ -293,10 +329,13 @@ const STYLES_SLATE_CARD = css`
...
+  border: 1px solid ${Constants.system.darkGray};
+>>>>>>> Stashed changes
```
Comment (jimmylee) [inline pages/index.js:338]: ✂️

File: pages/index.js [hunk 12]
```diff
@@ -293,10 +329,13 @@ const STYLES_SLATE_CARD = css`
   width: calc(100% / 3 + 1px);
```
Comment (jimmylee) [inline pages/index.js:329]: Why not 33.33%?

File: pages/index.js [hunk 13]
```diff
@@ -786,14 +827,32 @@ export default class IndexPage extends React.Component {
               </h3>
               <br />
               <br />
+=======
```
Comment (jimmylee) [inline pages/index.js:830]: ✂️

File: pages/index.js [hunk 14]
```diff
@@ -786,14 +827,32 @@ export default class IndexPage extends React.Component {
...
+                  for you, your files, and your friends.
+                </h1>
+
+                <div css={STYLES_ACTIONS_RIGHT}>
+                  <div css={STYLES_BUTTON_PRIMARY} onClick={() => window.open("/_")}>
+                    Use Slate
+                  </div>
+                </div>
+              </div>
+>>>>>>> Stashed changes
```
Comment (jimmylee) [inline pages/index.js:848]: ✂️

File: pages/index.js [hunk 15]
```diff
@@ -786,14 +827,32 @@ export default class IndexPage extends React.Component {
...
               <br />
-              <h2 css={STYLES_H2}>Create moodboards</h2>
-
               <video
                 css={STYLES_VIDEO_BIG}
                 autoPlay
                 loop
                 muted
+<<<<<<< Updated upstream
```
Comment (jimmylee) [inline pages/index.js:855]: ✂️

File: pages/index.js [hunk 16]
```diff
@@ -937,8 +996,431 @@ export default class IndexPage extends React.Component {
             <br />
             <br /> <div id="example-slates" css={STYLES_SLATE_CARD_GROUP}></div>
           </div>
+=======
```
Comment (jimmylee) [inline pages/index.js:999]: ✂️

File: pages/index.js [hunk 17]
```diff
@@ -937,8 +996,431 @@ export default class IndexPage extends React.Component {
...
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>-&gt;</div>
+                      </div>
+                    </div>
+                  </a>
+                </div>
+              </div>
+            </section>
+          </div>
+          <WebsitePrototypeFooter />
+>>>>>>> Stashed changes
```
Comment (jimmylee) [inline pages/index.js:1422]: ✂️

**Review by jimmylee — CHANGES_REQUESTED** [review]
Some things for you to look at! Be sure to test code.

File: pages/index.js (hunk 3 above)
Comment (tarafanlin) [reply pages/index.js:12]: yes I'm working on it

**Comment by jimmylee** [comment]
Screen Shot 2020-09-09 at 11 51 18 PM

Be sure to test a wider viewport. You can zoom out to catch this case.

File: pages/index.js [hunk 18]
```diff
@@ -434,511 +448,459 @@ export default class IndexPage extends React.Component {
...
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>@slate</div>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>-&gt;</div>
+                      </div>
+                    </div>
+                  </a>
+                </div>
+              </div>
+            </section>
           </div>
+          <WebsitePrototypeFooter />
```
Comment (jimmylee) [inline pages/index.js:902]: Why is <WebsitePrototypeFooter /> inside of this div?

File: pages/index.js [hunk 19]
```diff
@@ -434,511 +448,459 @@ export default class IndexPage extends React.Component {
   async componentDidMount() {
     const response = await Actions.health();
     console.log("HEALTH_CHECK", response);
-    this.addExampleSlates();
-  }
-
-  coolSlates = [
```
Comment (jimmylee) [inline pages/index.js:440]: Did you intend to erase this work that @gndclouds did?

**Review by jimmylee — CHANGES_REQUESTED** [review]
There is a visual bug you should address, and then I noticed a refactor that @gndclouds was taken out and replaced with a more manual solution again. Was that intentional? (Fine if it was!)

File: pages/index.js (hunk 19 above)
Comment (tarafanlin) [reply pages/index.js:440]: no I didn't. really sorry about this, I rebased this morning but not sure why I don't have this in my local repo. I will figure this out

File: pages/index.js (hunk 18 above)
Comment (tarafanlin) [reply pages/index.js:902]: this is something Martina suggested to fix the weird grey background thing. 
Did I not implement this correctly?

File: pages/index.js (hunk 19 above)
Comment (jimmylee) [reply pages/index.js:440]: No problem! Glad we've caught this now. Also if you view https://filecoin.onrender.com it looks like there was some other work.

One thing that is useful to check is outstanding pull requests, @gndclouds fixed a merge conflict that allowed @martinalong to merge after checking the work.

**Pull request closed without merging**